scripts/actions/mobskills/fuscous_ooze.lua

```diff
@@ -13,8 +13,11 @@ mobskillObject.onMobSkillCheck = function(target, mob, skill)
 end
 
 mobskillObject.onMobWeaponSkill = function(target, mob, skill)
-    -- TODO: Encumberance seems to do nothing?
-    xi.mobskills.mobStatusEffectMove(mob, target, xi.effect.WEIGHT, 50, 0, 45)
+    local power = math.random(1,16)
```

Comment:
`local power = math.random(1, 16)`

Need that space. Also, allign the power and duration while here.

Reply:
Formatted.